src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -112,6 +112,32 @@
     <None Remove="Views\Menu\TrustedPathView.xaml" />
     <None Remove="Views\GuidedTour\HtmlPages\Resources\ConnectTheNode.gif" />
   </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="HelixToolkit" Version="2.24.0" />
+    </ItemGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0-windows' ">
+        <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" /> 
+        <PackageReference Include="HelixToolkit.Core.Wpf" Version="2.24.0" />
+        <PackageReference Include="HelixToolkit.SharpDX.Core.Wpf" Version="2.24.0" />
```

Comment:
So are these replacements for `HelixToolkit.Wpf` and `HelixToolkit.Wpf.SharpDX` that are compatible with .net6?

Reply:
I think so https://www.nuget.org/packages/HelixToolkit.SharpDX.Core.Wpf/